src/App.tsx

```diff
@@ -15,37 +17,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SetByReverse = true;
+
+enum SortedFields {
```

Comment:
The enum 'SortedFields' contains a value 'SetByDanger' which is not used or defined in the sorting logic. Ensure that all enum values have a clear purpose and are used within the application logic, or remove unused values to avoid confusion.